core/ir/aarch64/codec.py

```diff
@@ -390,6 +390,42 @@ def generate_opcode_names(patterns):
           '#endif /* OPCODE_NAMES_H */']
     return '\n'.join(c) + '\n'
 
+# Generates pairs of opcodes and masks for their operands for all the
+# side-effect-free and non-branch instructions suppported by the decoder.
+# The generated file is used by the drstatecmp-fuzz-app.
+def generate_opcode_opnd_pairs(patterns):
+    c = ['#ifndef OPCODE_OPND_PAIRS_H',
+         '#define OPCODE_OPND_PAIRS_H 1',
+         '',
+         '#include <stdint.h>',
+         '',
+         'typedef struct {',
+         '  uint32_t opcode;',
+         '  uint32_t opnd;',
+         '} opcode_opnd_pair_t;',
```

Comment:
suggest: use a dr_ (and DR_ for `#define DR_FUZZ_INST_CNT`) prefix for all global types and variables to avoid namespace collisions.  We do have some legacy types that don't have a prefix, but going forward we should try to avoid problems.  We have had a number of collision issues in the past with statically linking DR into large applications that also have other third-party C libraries linked in.

Reply:
Done.